devicon.json

```
@@ -1984,7 +1984,8 @@
         "tags": [
             "javascript",
             "rust",
-            "language"
+            "language",
```

Should we be keeping this tag here? 🤔
I think it fits more when the technology is actually a language, like JavaScript or Rust, but not when it uses some language like this one, React or some other Framework.
Let me know what you guys think. 🙂
```suggestion
            "language",
```

---

I agree. The `language` tag doesn't quite fit `deno` and `nodejs`.